gallery/src/components/NxTextInput/NxTextInputPage.tsx

```diff
@@ -192,6 +192,33 @@ const NxTextInputPage = () =>
           </NxTable.Row>
         </NxTable.Body>
       </NxTable>
+      <NxStatefulAccordion defaultOpen={false}>
+        <NxAccordion.Header>
+          <h2 className="nx-accordion__header-title">Regarding &lt;input type="number"&gt;</h2>
```

Comment:
Makes me realize we should probably add a component for this

Reply:
> Makes me realize we should probably add a component for accordion titles

https://issues.sonatype.org/browse/RSC-728